src/dags/winkelgebieden.py

```diff
@@ -108,15 +133,15 @@ def quote(instr):
 
 dag.doc_md = """
     #### DAG summery
-    This DAG containts permit announcements (bekendmakingen)
+    This DAG containts shopping area's (winkelgebieden)
     #### Mission Critical
     Classified as 2 (beschikbaarheid [range: 1,2,3])
     #### On Failure Actions
     Fix issues and rerun dag on working days
     #### Point of Contact
     Inform the businessowner at [businessowner]@amsterdam.nl
     #### Business Use Case / process
-    Permit allowance (vergunningverlening)
+    Na
     #### Prerequisites/Dependencies/Resourcing
-    None
+    https://api.data.amsterdam.nl/v1/winkelgebieden/winkelgebieden/
```

Comment:
Zou hier nog iets bij moeten over waar het oorspronkelijke input bestand vandaan komt?

Reply:
Ja lijkt mij prima. Ik weet alleen niet wat ik daar in moet zetten. Weet jij toevallig wie dit aanlevert en wat de bron is? Maar misschien voor nu een idee om deze markdown versie definitief te maken in de groep. Ik zal er een post aan wijden in Slack.